src/libs/actions/Report.js

```diff
@@ -1213,45 +1247,89 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
 
     // Do not autolink if someone explicitly tries to remove a link from message.
     // https://github.com/Expensify/App/issues/9090
-    const htmlForNewComment = parser.replace(textForNewComment, {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
+    const htmlForEditedComment = parser.replace(textForNewComment, {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
+    const textForEditedComment = parser.htmlToText(htmlForEditedComment);
```

Comment:
Any reason we are converting the html to text instead of just using `textForNewComment`?

Reply:
**tip:** if you can avoid changing any variable names it will make your changes easier to follow. In this case, I don't think "new" vs "edited" is really worth changing.

Reply:
No reason at all, thx for catching that! Using the parameter now instead of creating the variable and returned to the previous var name.